lib/shared/puppet_forge/version.rb

```diff
@@ -0,0 +1,3 @@
+module PuppetForge
+  VERSION = '2.0.0dev'
```

Comment:
Gem versioning doesn't allow a hyphen between the patch release and the prerelease but I can do `2.0.0alpha`